src/_guides/language/analysis-options.md

```diff
@@ -67,30 +69,34 @@ at the root of the package, in the same directory as the pubspec file.
 Here's a sample analysis options file:
 
 {% prettify yaml %}
-analyzer:
-  strong-mode:
-    implicit-casts: false
-  errors:
-    todo: ignore
-  exclude:
-    - flutter/**
-    - lib/api/*.dart
+include: package:pedantic/analysis_options.yaml
 
 linter:
   rules:
-    - avoid_empty_else
-    - cancel_subscriptions
-    - close_sinks
-    - unnecessary_const
-    - unnecessary_new
+    - camel_case_types
+
+analyzer:
+#   exclude:
+#     - path/to/excluded/files/**
 {% endprettify %}
 
-YAML is sensitive to whitespace&mdash;don't use tabs in a YAML file,
-and use 2 spaces to denote each level of indentation.
+The <code>include: <em>url</em></code> entry
+brings in YAML code from the specified URL — in this case,
+from a file in the `pedantic` package.
+The `linter:` entry [enables linter rules](#enabling-linter-rules).
+You can use the `analyzer:` entry to customize static analysis —
+[enabling stricter type checks](#enabling-additional-type-checks), 
+[excluding files](#excluding-files),
+suppressing specific rules [for a file](#suppressing-rules-for-a-file) or
+[for a line of code](#suppressing-rules-for-a-line-of-code),
```

Comment:
Users can't actually suppress a rule for either a file or a single line from within the analysis options file. Doing so requires adding 'ignore' comments to a source file.

Reply:
Good catch.  This would still be great to include but clarifying that this is done in source and not options is a good idea.

Reply:
OK, I'll clarify that. Good catch!